OutOfSchool/OutOfSchool.WebApi.Tests/Services/ApplicationSensitiveServiceTests.cs

```diff
@@ -0,0 +1,360 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Threading.Tasks;
+using AutoMapper;
+using FluentAssertions;
+using Microsoft.Extensions.Localization;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using MockQueryable.Moq;
+using Moq;
+using NUnit.Framework;
+using OutOfSchool.Common.Enums;
+using OutOfSchool.Common.Models;
+using OutOfSchool.EmailSender;
+using OutOfSchool.RazorTemplatesData.Services;
+using OutOfSchool.Services.Enums;
+using OutOfSchool.Services.Models;
+using OutOfSchool.Services.Models.SubordinationStructure;
+using OutOfSchool.Services.Repository;
+using OutOfSchool.Tests.Common;
+using OutOfSchool.WebApi.Config;
+using OutOfSchool.WebApi.Models;
+using OutOfSchool.WebApi.Models.Application;
+using OutOfSchool.WebApi.Models.Workshops;
+using OutOfSchool.WebApi.Services;
+namespace OutOfSchool.WebApi.Tests.Services;
+
+[TestFixture]
+public class ApplicationSensitiveServiceTests
+{
+    private const int N = 1;
+    private ISensitiveApplicationService service;
+    private ISensitiveApplicationService sensitiveApplicationService;
```

Comment:
(Redundant code) `sensitiveApplicationService` field is not used

Reply:
> (Redundant code) `sensitiveApplicationService` field is not used

Done